packages/devtools_app/lib/src/profiler/cpu_profile_controller.dart

```diff
@@ -109,9 +130,40 @@ class CpuProfilerController with SearchControllerMixin<CpuStackFrame> {
     return matches;
   }
 
-  void loadData(CpuProfileData data) {
+  void loadProcessedData(CpuProfileData data) {
     assert(data.processed);
     _dataNotifier.value = data;
+    _dataByTag[userTagNone] = data;
+  }
+
+  Future<void> loadDataWithTag(String tag) async {
+    _userTagFilter.value = tag;
+
+    _dataNotifier.value = null;
+    _processingNotifier.value = true;
+
+    try {
+      assert(false);
```

Comment:
I'm guessing this isn't supposed to be here?

Reply:
you are correct, removed